openfl/interface/interactive_api/experiment.py

```diff
@@ -143,15 +133,22 @@ def prepare_workspace_distribution(
         # Compress te workspace to restore it on collaborator
         self.arch_path = self._pack_the_workspace()
 
-        # DO CERTIFICATES exchange
-
     def start(self, *, model_provider, task_keeper, data_loader,
-              rounds_to_train, delta_updates=False, opt_treatment='RESET'):
+              rounds_to_train, delta_updates=False, opt_treatment='RESET',
+              device_assignment_policy='CPU_ONLY'):
```

Comment:
Can we use `DevicePolicy.CPU_ONLY` here?

Reply:
yes. I created an issue for this